### DISCUSSION TITLE
Add Wall Socket Support?

### DISCUSSION THREAD

**Discussion by knebb** [netbox-community/netbox#14786] [Ideas]
Hi,

I realized I can connect a device (ie switch interface) to the front ports of a patchpanel which will be forwarded to the rear port of the patchpanel. And then connect the rear port to a device interface.

But in physical world the "rear cable" of the patchpanel goes trough a cable to a wall box. Then I am plugging in the cable from the device's interface.

So I would like to to be able to link a rear cable to a new type of device- to a wall box (with front and rear cable, too).

Does this make sense?

/KNEBB

**Comment by candlerb** [comment]
Yes you can just model the wallbox as a "device" with say 1 , 2 or 4 front ports (the jacks) and an equal number of rear ports. Then connect the rear port(s) on your patchpanel to the rear port(s) of the wallbox.

The room which contains the wallbox can be modelled simply as a "location" - you don't need a "rack" for it.

(For my home network I don't bother with this: I just create a cable from the device's interface to the appropriate patchpanel rearport when it gets connected. It's a matter of taste and whether you get value from the more detailed modelling)

**Reply by knebb** [reply]
Yes,
I expected this. When writing the above I had the same idea. So working fine, indeed.

Thanks!

Can be closed.
/KNEBB
